engines/bastion_katello/app/assets/javascripts/bastion_katello/products/discovery/views/docker-discovery.html

```diff
@@ -0,0 +1,84 @@
+<span page-title>{{ 'Docker Repository Discovery' | translate }}</span>
+
+<div bst-alerts success-messages="successMessages" error-messages="errorMessages"></div>
+
+<div class="row">
+  <form ng-submit="discover()" name="discoveryForm" class="col-sm-6">
+    <input type="text"
```

Comment:
Putting this into a `<div class="form-group">` will fix the bottom padding of the input.